README.md

```diff
@@ -187,3 +188,5 @@ This project exists thanks to all the [contributors]((https://github.com/ZupIT/h
 [Semgrep]: https://github.com/returntocorp/semgrep
 [EsLint]: https://github.com/eslint/eslint
 [Flawfinder]: https://github.com/david-a-wheeler/flawfinder
+[Flawfinder]: https://github.com/david-a-wheeler/flawfinder
+[PHPCS]: https://github.com/FloeDesignTechnologies/phpcs-security-audit
```

Comment:
Flawfinder duplicated.
Please add Flawfinder and PhpCS in [start.go](https://github.com/ZupIT/horusec/blob/feature/phpcs/horusec-cli/cmd/horusec/start/start.go#L213) file to doc in CLI

Reply:
Fixed!